### PR TITLE
Added a mini parser to parse the formated Nebula Value in CI

### PR DESCRIPTION
**Pull request by dutor**
To test this PR:
 * `pip install behave`
 * `pip install ply`
 * `behave tests/tck`

TODO: Use `Value`s from nebula-python, instead of redefining them along with the parser

### CODE REVIEW

File: tests/tck/features/utils.feature [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        | "string'substr'"                                      | str       |
+        | 'string"substr"'                                      | str       |
+        | []                                                    | list      |
+        | [1,2,3]                                               | list      |
+        | [-[:e1{}]-,<-[:e2{}]-,-[:e3{}]->]                     | list      |
+        | {1,2,3}                                               | set       |
+        | {}                                                    | dict      |
+        | {k1: 1, k2:true}                                      | dict      |
+        | ('vid':t{p1:0,p2:' '})                                | Node      |
+        | -[:e{p1:0,p2:true}]-                                  | Edge      |
```
Comment (yixinglu) [inline tests/tck/features/utils.feature:21]: support edge rank, maybe the format is like:

```text
-[:e@0{p1: 0, p2: ''}]->
```

Comment (dutor) [reply tests/tck/features/utils.feature:23]: Forgot that.
Addressed.

**Review by yixinglu — DISMISSED** [review]
Excellent

File: tests/tck/steps/environment.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from behave import *
```
Comment (laura-ding) [inline tests/tck/steps/environment.py:1]: add head desc

File: tests/tck/steps/value-parsing.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import os
```
Comment (laura-ding) [inline tests/tck/steps/value-parsing.py:1]: ditto

File: tests/tck/utils/utils.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+def parse(input):
+    if isinstance(input, bh.Table):
+        return parseTable(input)
+    elif isinstance(input, bh.Row):
+        return parseRow(input)
+    else:
+        raise Exception('Unable to parse %s' % str(input))
+
+def parseTable(table):
```
Comment (yixinglu) [inline tests/tck/utils/utils.py:19]: parse_table

File: tests/tck/utils/utils.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    names = table.headings
+    rows = []
+    for row in table:
+        rows.append(parseRow(row))
+    return DataSet(column_names = table.headings, rows = rows)
+
+def parseRow(row):
```
Comment (yixinglu) [inline tests/tck/utils/utils.py:26]: parse_row

File: tests/tck/utils/utils.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2020 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+#
+
+from nebula2.common.ttypes import Value,Vertex,Edge,Path,Step,NullType,DataSet,Row,List
+import nbv
+from behave import model as bh
```
Comment (yixinglu) [inline tests/tck/utils/utils.py:9]: maybe we need to consider whether to use pytest-bdd as our backend implementation

**Review by yixinglu — APPROVED** [review]

**Review by laura-ding — APPROVED** [review]

**Pull request merged**